pkgs/top-level/all-packages.nix

```diff
@@ -5569,9 +5569,18 @@ with pkgs;
 
   glslang = callPackage ../development/compilers/glslang { };
 
-  go_bootstrap = callPackage ../development/compilers/go/1.4.nix {
-    inherit (darwin.apple_sdk.frameworks) Security;
-  };
+  go_bootstrap = if stdenv.isAarch64 then
+    srcOnly {
+      name = "go-1.8-linux-arm64-bootstrap";
+      src = fetchurl {
+        url = "https://cache.xor.us/go-1.8-linux-arm64-bootstrap.tar.xz";
```

Comment:
* `cache.xor.us` now presents the cert for `blog.xor.us`, breaking TLS
* even if you allow certificate mismatch, URL returns 404